README.md

```diff
@@ -1,2 +1,60 @@
-# executor-j5s
-Jenkins Executor for Screwdriver
+# Screwdriver Jenkins Executor
+
+This executor plugin extends the [executor-base-class], and provides methods to start jobs from Jenkins
+
+## Usage
+
+```bash
+npm install screwdriver-executor-j5s
+```
+
+### Configure
+The class provides a couple options that are configurable in the instantiation of this Executor
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config        | Object | Configuration Object |
+| config.username | String | The username for Jenkins cluster  |
+| config.password | String | The password or token for Jenkins cluster  |
+| config.host | String | The hostname for the Jenkins cluster |
+
+### Start
+The `_start` method takes advantage of the input validation defined in the [executor-base-class].
```

Comment:
_start was from the base executor class and we implement the detail of that in this executor.